src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -341,6 +341,13 @@ private protected static LanguageServerProtocol GetLanguageServer(Solution solut
             return workspace.ExportProvider.GetExportedValue<LanguageServerProtocol>();
         }
 
+        private protected static RequestExecutionQueue GetRequestQueue(Solution solution)
```

Comment:
```suggestion
        private protected static RequestExecutionQueue CreateRequestQueue(Solution solution)
```

The name would have implied that if called multiple times it'd get the same queue; I worry that the the tests could have some 'surprises' if somebody thought that'd be the case.